src/features/data-streams/common/gettingStarted.mdx

```diff
@@ -26,7 +26,7 @@ Read data from a Data Stream and validate the answer on-chain. This example uses
 
 ## Deploy the Chainlink Automation upkeep
```

Comment:
This step needs to have the constructor params passed.  That is incorrectly sitting under the Log Emitter instructions now.

Also the constructor now accepts only a single param but this doc (under the log emitter section) instructs the user to pass in 2.

Reply:
Fixed in a previous PR